index.html

```diff
@@ -170,6 +170,43 @@ <h2>
           readonly attribute FrozenArray&lt;GamepadButton&gt; buttons;
         };
       </pre>
+      <p>
+        Instances of {{Navigator}} are created with the internal slots described in the following table:
+      </p>
+      <table class="simple" dfn-for="Navigator" dfn-type="attribute">
+        <tr>
+          <th>Internal slot
+          <th>Initial value
+          <th>Description (non-normative)
+        <tr>
+          <td><dfn>[[\exposed]]</dfn>
+          <td>`false`
+          <td>A flag indicating that the {{Gamepad}} object has been exposed to script.
+        <tr>
+          <td><dfn>[[\axisMapping]]</dfn>
+          <td>`{}`
+          <td>Mapping from unmapped axis index to an index in the {{Gamepad/axes}} array
+        <tr>
+          <td><dfn>[[\axisMinimums]]</dfn>
+          <td>`null`
+          <td>An array containing the minimum logical value for each axis.
+        <tr>
+          <td><dfn>[[\axisMaximums]]</dfn>
+          <td>`null`
+          <td>An array containing the maximum logical value for each axis.
```

Comment:
Probably want a list or [=ordered set=] here... 

```suggestion
          <td>A [=list=] containing the maximum logical value for each axis.
```

Same for the rest that use "array"...